repos/system_upgrade/cloudlinux/actors/handleelspackages/files/els_script_handler.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+
+BACKUP="/etc/cl-elevate-saved"
+RUNNING_KERNEL=$(uname -r)
+
+echo "Starting ELS package handling"
+echo "Running kernel: ${RUNNING_KERNEL}"
+
+mkdir -p "${BACKUP}" 2>/dev/null || {
+    echo "Failed to create backup directory ${BACKUP}"
+    exit 1
+}
+
+# Check if ELS repositories are present
+if ! ls /etc/yum.repos.d/centos*els*.repo &> /dev/null; then
+    echo "No ELS repositories found, skipping processing"
+    exit 0
+fi
+
+echo "ELS repositories detected. Processing ELS packages before conversion..."
+
+# Create package backup list
+rpm -qa | sort > "${BACKUP}/els_packages_backup_$(date +%Y%m%d_%H%M%S).txt" || {
+    echo "Warning: Failed to create package backup list"
+}
+
+# Function to check if kernel is in use
+is_kernel_in_use() {
+    local kernel_pkg=$1
+    local kernel_ver=$(rpm -q --queryformat '%{VERSION}-%{RELEASE}' "$kernel_pkg")
+    [[ "$kernel_ver" == "$RUNNING_KERNEL" ]]
+}
+
+# Prevent ELS packages from being reinstalled
+if ! grep -q "exclude=\*.tuxcare.els\*" /etc/yum.conf; then
+    {
+        echo
+        echo "# Added by cloudlinux Elevate"
+        echo "exclude=*.tuxcare.els*"
+    } >> /etc/yum.conf || echo "Warning: Failed to add exclusion to yum.conf"
+fi
+
+# Save and disable ELS repos
+cp /etc/yum.repos.d/centos*els*.repo "${BACKUP}/" 2>&1
+echo "Disabling ELS repositories..."
+yum-config-manager --disable centos7-els 2>&1
+for i in {1..6}; do
```

Comment:
ELS team can easily add more rollout slots in the future, let's change this to some proper search or centos7els-rollout-* if config manager supports that.